lua/nvim-web-devicons.lua

```diff
@@ -801,13 +801,13 @@ local icons = {
     name = "Material",
   },
   ["md"] = {
-    icon = "",
-    color = "#519aba",
-    cterm_color = "67",
+    icon = "",
+    color = "#ffffff",
+    cterm_color = "white",
     name = "Md",
   },
   ["mdx"] = {
-    icon = "",
+    icon = "",
     color = "#519aba",
     cterm_color = "67",
```

Comment:
colors don't match, why did you switch to full white ?

Reply:
<img width="128" src="https://user-images.githubusercontent.com/42694704/177314068-0c5b0f27-fac3-4108-8826-2387fda10115.png" />

It's because markdown's original color is full white